src/chapters/classes/methods.rst

```diff
@@ -0,0 +1,127 @@
+Methods
+=======
+
+Calling Methods on Objects
+--------------------------
+
+A **method** is a procedure that belongs to a class. In Java, all
```

Comment:
is it inaccurate to replace procedure with function? if not, id do that, we touch briefly on this in the intro to data types section, and mention we will return to this idea later, so it could be good to link back to that part here. ill go searching for it

Reply:
I replaced it! I think since people will be coming into this with some knowledge of coding, it will make sense. Thanks!